CASE-STUDIES-AND-EXPERIENCES.md

```diff
@@ -5,23 +5,16 @@ title: Case Studies and Experiences
 site_side: true
 tags: security-qualitative-metrics
 project: true
-level: 4
 type: documentation
 pitch: The OWASP Security Qualitative Metrics is the most detailed list of metrics which evaluate security level of web projects. It shows the level of coverage of OWASP ASVS. 
 ---
 
+# ![Project Logo](assets/images/logo3_small.png) OWASP Security Qualitative Metrics
 
-
-# ![Project Logo](images/logo3_small.png) Case Studies and Experiences 
-# [![OWASP Flagship](https://img.shields.io/badge/owasp-flagship-blue.svg)](https://owasp.org/projects/)
```

Comment:
Your project isnt a flagship

Reply:
Why do you change it now. Initially, when it is prepared it was reviewed and they told me everything was ok. I didn't assign it to be flagship in the first place. 

It is very discouraging to make a change at this step. I do not want this change if possible.

Reply:
It was listed in the flagship projects when it is accepted and this is not something I made. I do not want to accept to make a change in this step. This should be discussed if necessary and should not be decided as a part of a web site format change update.

Reply:
You left defaults and didn't bother reading the on boarding info (as far as I can see)

Reply:
OWASP staff already pushed a change for index.md correcting the project level. This is further to that fix.

Reply:
"You left defaults and didn't bother reading the on boarding info (as far as I can see)" this is not correct. This is just an assumption. I am not familiar with the MD format, so I had some difficulties, to do some things, like version but I didn't leave anything in the defaults.